x-pack/plugins/ml/public/application/components/data_grid/feature_importance/decision_path_chart.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// adjust the height so it's compact for items with more features
```

Comment:
should this comment go further down where it's relevant or can it be removed?

Reply:
Updated here https://github.com/elastic/kibana/pull/73561/commits/ef815d11a2bf334d703366daff3519b38b848e11